common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -14,10 +14,9 @@
 
 define(
 'video/01_initialize.js',
-['video/03_video_player.js', 'video/00_i18n.js'],
+['video/03_video_player.js', 'video/00_i18n.js', 'moment'],
 function (VideoPlayer, i18n) {
     var moment = window.moment;
```

Comment:
Can you add ``moment`` as a third parameter and use it directly? Does it get shimmed so that it returns itself? That would be cleaner than using it via ``window``.